nixos/tests/c2FmZQ.nix

```
@@ -0,0 +1,77 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "c2FmZQ";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ hmenke ];
+  };
```

Maybe it would be better to import lib?

```suggestion
import ./make-test-python.nix ({ pkgs, lib, ... }: {
  name = "c2FmZQ";
  meta.maintainers = with lib.maintainers; [ hmenke ];
```